library/spit_results.py

```diff
@@ -41,6 +42,71 @@ def __init__(self, module):
         self.vals = module.params['vals']
         self.fact_names = module.params['fact_names']
 
+    # pylint: disable=too-many-instance-attributes
+    class PkgInfo(object):
```

Comment:
I think this class is code from a different PR that is in here because we haven't merged the other one yet. Is that right? It still looks good to me.